webapp/src/app/components/wheel/wheel.component.scss

```diff
@@ -6,7 +6,7 @@
     margin: 24px;
     border: 1px solid #666;
     overflow: hidden;
-    width: 400px;
+    width: 300px;
```

Comment:
Maybe consider putting this in a variable if it's duplicated

Reply:
you may want to change one or the other at some point so best to be uncoupled